### ISSUE TITLE
message.html

### ISSUE THREAD

**Issue by LeidyKCastaneda** [LeidyKCastaneda/Reto2#3]
<!DOCTYPE html>
<html>
<head>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>    
    <script type="text/javascript" src="./message.js"></script>
    <title>Reto2</title>
</head>
<body> 
    <h1>Registro de mensajes</h1>
    <p id="texto"></p>
    <div id="Mensajes"></div>
        <label> ID mensaje</label>
        <input type="number" placeholder=" " id="ID" /><br>
        <label> Escriba su mensaje </label>
        <input type="text" placeholder=" " id="MESSAGETEXT" /><br>  
        <button onclick="consultarmessage()">Consultar</button> 
        <button onclick="createmessage()">Enviar</button>
        <button onclick="editarMessage()">Editar</button>

        
    </div>

    
</body>
</html>
